static/js/components/Toast.js

```diff
@@ -13,24 +12,24 @@ export default class Toast extends React.Component {
   };
 
   componentDidMount() {
-    const { onTimeout, open, timeout } = this.props;
+    const { onTimeout, timeout } = this.props;
 
-    if (open && onTimeout) {
+    if (onTimeout) {
       setTimeout(onTimeout, timeout);
     }
   }
 
-  componentDidUpdate(prevProps: Object) {
+  componentDidUpdate() {
     const { onTimeout, timeout } = this.props;
-    if (!prevProps.open && this.props.open && onTimeout) {
+    if (onTimeout) {
       setTimeout(onTimeout, timeout);
     }
```

Comment:
I don't believe so. Any case in the past where `open` was `false`, now this component simply won't be rendered, so the code won't be called at all. But if you can think of a way that `setTimeout` might be called multiple times, let me know.